src/services/config/config-service.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from "node:fs"
+import * as path from "node:path"
+import { CLIError } from "@oclif/core/lib/errors"
+
+const DEFAULT_CONFIG_PATH = path.resolve(__dirname)
+const DEFAULT_CONFIG_FILE = "default-config.json"
+const CONFIG_FOLDER = ".ent"
+const CONFIG_FILE = "config.json"
+
+export default class ConfigService {
+  private readonly _config: any
+
+  constructor(filePath = DEFAULT_CONFIG_PATH) {
+    this._config = this.readConfigFile(filePath)
+  }
+
+  get config(): any {
+    return this._config
+  }
+
+  readConfigFile = (filePath: string): any => {
+    const templateFileContent = fs.readFileSync(
+      path.join(filePath, DEFAULT_CONFIG_FILE),
+      {
+        encoding: "utf-8"
+      }
+    )
+
+    return JSON.parse(templateFileContent)
+  }
+
+  writeConfigFile = (bundlePath: string, fileFolder = CONFIG_FOLDER): void => {
```

Comment:
there should be the following methods:
initConfigFile(path)
readConfigFile()
writeConfigFile()